stdlib/scripts/run-benchmarks-sequentially.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+##===----------------------------------------------------------------------===##
+# Copyright (c) 2024, Modular Inc. All rights reserved.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions:
+# https://llvm.org/LICENSE.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##===----------------------------------------------------------------------===##
+
+set -euo pipefail
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+REPO_ROOT="${SCRIPT_DIR}"/../..
+BUILD_DIR="${REPO_ROOT}"/build
+
+mkdir -p "${BUILD_DIR}"
+
+source "${SCRIPT_DIR}"/build-stdlib.sh
+
+TEST_UTILS_PATH="${REPO_ROOT}/stdlib/test/test_utils"
+mojo package "${TEST_UTILS_PATH}" -o "${BUILD_DIR}/test_utils.mojopkg"
+
+BENCHMARK_PATH="${REPO_ROOT}/stdlib/benchmarks"
+if [[ $# -gt 0 ]]; then
+  # If an argument is provided, use it as the specific file or directory.
+  BENCHMARK_PATH=$1
+fi
+
+# Run the benchmarks
+for f in $(find $BENCHMARK_PATH | grep -E ".mojo|.🔥"); do 
+  echo "-------------------- BENCHMARK START ${f} --------------------"
+  mojo run -I build/ $f
```

Comment:
This is fine to JIT them (i.e. what `mojo run` does), did you verify whether `-I build` will pick up your changes in local `stdlib.mojopkg`?  I think this may "mostly work" modulo the fact that the `benchmark.mojopkg` will still be the one picked up from your installation/not your local build and so _that package_ won't take into account your recent modifications to the library.  This is inevitable right now since the source is not shipped for the `benchmark` module, and thus cannot be re-compiled on an external developer's machine using the new `stdlib` sources.  In most cases, this will be "fine" except for breaking changes (e.g. an interface changed and you're effectively mixing ABIs at that point via different mojopkgs).